include/curl-session.h

```diff
@@ -78,7 +78,8 @@ class CurlSession : public CurlCallbacks
     }
 
 
-    void addFormFile(const std::string& key, const std::string& filePath, const std::string& mimeType);
+    void addFormFile(const char* key, const char* filePath, const char* mimeType);
```

Comment:
I've come with a solution that both keeps C++ style and avoids creating temporary std::string object in case we have C string.